src/Text/Pandoc/Writers/Docx.hs

```diff
@@ -175,9 +175,19 @@ writeDocx opts doc = do
                    [] -> stTocTitle defaultWriterState
                    ls -> ls
 
+  let lofTitle = case lookupMetaInlines "lof-title" meta of
+                   [] -> stLofTitle defaultWriterState
+                   ls -> ls
+
+  let lotTitle = case lookupMetaInlines "lot-title" meta of
+                   [] -> stLotTitle defaultWriterState
+                   ls -> ls
+
```

Comment:
Instead of this, I'd recommend just using the Translations interface.
```
lofTitle <- B.text <$> translateTerm Translations.ListOfFigures
lotTitle <- B.text <$> translateTerm Translations.ListOfTables
```

Reply:
I note that we currently hard-code  the string "Table of Contents."  We should also use Translations for that.

Reply:
I ended up getting the following error:
```
src/Text/Pandoc/Writers/Docx/OpenXML.hs:151:15: error:
    Not in scope: ‘B.text’
    No module named ‘B’ is imported.
    |
151 |   tocTitle <- B.text <$> translateTerm Translations.TableOfContents
    |               ^^^^^^

src/Text/Pandoc/Writers/Docx/OpenXML.hs:151:40: error:
    Not in scope: data constructor ‘Translations.TableOfContents’
    No module named ‘Translations’ is imported.
    |
151 |   tocTitle <- B.text <$> translateTerm Translations.TableOfContents
    |                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
I'm sorry @jgm if you could guide me on the API for `Translations` I'd much appreciate it.

Reply:
Maybe `B.` isn't needed.  It's Text.Pandoc.Builder; some modules import this qualified as `B`, but maybe not the Docx writer. Check the imports.

https://hackage.haskell.org/package/pandoc-3.3/docs/Text-Pandoc-Translations.html
https://hackage.haskell.org/package/pandoc-3.3/docs/Text-Pandoc-Translations-Types.html

Reply:
We already do `setTranslations` at the beginning of the docx writer.
All you need is `translateTerm` and the various terms (in T.P.Translations.Types).

Reply:
Actually, you probably don't need `B.text` at all.
Eventually you just want a string `Text` with "Table of Figures" or whatever, to put into the relevant XML attribute.  `translateTerm` will give you that.  `B.text` is only needed if you want to convert that to an `Inlines`.  (I used it because you had used `lookupMetaInlines` which gives an `Inlines`.)

Reply:
Tried without `Text.Pandoc.Builder` and it gave an error mentioning requiring it to be a `Inline` instead of a `Text`.

Reply:
yes, sorry, this was an incorrect suggestion on my part; I hadn't seen the whole context.

Reply:
To convert an Inlines to [Inline], you just need B.toList.

Reply:
Looks like that did the trick! Thanks so much @jgm !